test/onnx/gen_onnx.py

```diff
@@ -2013,6 +2012,14 @@ def neg_test():
 
     return ([node], [x], [y])
 
+@onnx_test
+def not_test():
+    x = helper.make_tensor_value_info('0', TensorProto.INT32, [4])
+    y = helper.make_tensor_value_info('1', TensorProto.INT32, [4])
```

Comment:
The input here should be BOOL type

Reply:
Why does it have to be BOOL? Unary not should work on any data type, what am I missing?

Reply:
That is specified by the onnx operator. You can keep this test and one for the BOOL type.

Reply:
Got it! Thanks Shucai.